test/sqllogictest/explain-pushdown.slt

```diff
@@ -95,6 +95,16 @@ EXPLAIN FILTER PUSHDOWN FOR SELECT * FROM jsonb_fields where timestamp > 1000 AN
 ----
 materialize.public.jsonb_fields  2011  0  2  0
 
+# EXPLAIN FILTER PUSHDOWN FOR MATERIALIZED VIEW is also supported
+
+statement ok
+CREATE MATERIALIZED VIEW big_numbers AS SELECT * FROM numbers WHERE value > 10000;
+
+query TIIII
+EXPLAIN FILTER PUSHDOWN FOR MATERIALIZED VIEW big_numbers
+----
+materialize.public.numbers  0  0  0  0
+
```

Comment:
You could maybe add some more tests. For example, you could call it on all the MVs in `materialized_views.slt`. You can get a list of MV names in that file by looking at the output of the second to last introspection query. If it's hard to check the concrete output you could just do `statement ok` on them.

Reply:
Yeah, the output for (non-empty) materialized views is not necessarily stable, since they reveal the exact part splits / byte counts which may change. But I can do `statement ok`!